### ISSUE TITLE
HasInlineMorphToFields trait prevents other relationships fields to be displayed

### ISSUE THREAD

**Issue by fntneves** [dcasia/nova-inline-morph-to#5]
Hello.

I'm facing a new issue, described as follows:

Imagine that there's another relationship field in the Article resource:
`MorphMany::make('Collection')`

The details page, when resource uses the HasInlineMorphToFields trait, is completely replaced by one that does not show this new relationship field.

Another issue I've found is that labels are not used to display the morph types. (singularLabel and pluralLabel). Instead, I'm seeing the actual Resource class.

See the following screens, before and after using this package.
<img width="1407" alt="Screenshot 2019-10-03 at 21 26 29" src="https://user-images.githubusercontent.com/1722352/66161617-9e445400-e624-11e9-8869-bf280f9164c9.png">

<img width="1413" alt="Screenshot 2019-10-03 at 21 26 51" src="https://user-images.githubusercontent.com/1722352/66161588-8bca1a80-e624-11e9-9b7a-ca64a8db4d6f.png">

**Comment by milewski** [comment]
Hi @fntneves , 

> Another issue I've found is that labels are not used to display the morph types. (singularLabel and pluralLabel). Instead, I'm seeing the actual Resource class.

Based on this I assume you are using an old version where the label and the mentioned issues on the detail page were still a problem. 

Can you double check that you are using version [v0.1.6](https://github.com/dcasia/nova-inline-morph-to/releases/tag/v0.1.6)

**Comment by fntneves** [comment]
I was using 0.1.4, actually.

This is now fixed. This package is getting better, you're doing a great job!

Additionally, is it possible to leverage "displayUsingLabels" function on Select fields?
Select option values are being displayed instead of labels.

```php
Select::make('Unidade do valor de desconto', 'discount_unit')
    ->options([
        DiscountUnitType::FIXED => 'Fixo',
        DiscountUnitType::PERCENTAGE => 'Percentual',
     ])
    ->displayUsingLabels(),
```

I've attached two more screenshots:
<img width="673" alt="Screenshot 2019-10-04 at 08 06 27" src="https://user-images.githubusercontent.com/1722352/66188067-489c9580-e67e-11e9-8c50-4ebab668a867.png">
<img width="521" alt="Screenshot 2019-10-04 at 08 06 32" src="https://user-images.githubusercontent.com/1722352/66188068-489c9580-e67e-11e9-99f9-3c7ca1dd73da.png">
